doc/maintaining/installing/install-from-package.rst

```diff
@@ -109,6 +109,10 @@ CKAN:
    change the :ref:`sqlalchemy.url` setting in your
    |production.ini| file to reference your |postgres| server.
 
+.. note::
+     
+   The commands mentioned below are tested for Ubuntu system
```

Comment:
What's the point of continuous repetition if Ubuntu was mentioned as requirement in the very first paragraph of this page?

Reply:
Yes, Ubuntu has mentioned at the top, but still One issue has been raised by @wood-chris while he was trying to install on windows. 
If we add Ubuntu explicitly with the command so in this way it will be more clear for end user perspective